portfolio/src/main/java/com/google/sps/Constants.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps;
+
+//Class to store constants
+public final class Constants {
+  public static final String _COMMENT  = "Comment";
+
+  public static final String AUTHOR = "author";
+
+  public static final String COMMENT = "comment";
+
+  public static final String TIMESTAMP = "timestamp";
+
+  public static final int COMMENT_MIN_LENGTH = 4;
+
+  public static final int COMMENT_MAX_LENGTH = 250;
+
+  public static final String AUTHOR_EMPTY_ERROR = "Author name cannot be empty.";
+
+  public static final String COMMENT_LENGTH_ERROR = "Comment should be between " + Integer.toString(COMMENT_MIN_LENGTH)
```

Comment:
You can leave this string in the file, as it is an error message.
Prefer using String Builder. It helps reduce memory usage

Reply:
Strings in JAVA are immutable, hence every '+' sign results in a new string object. String Builder uses a single object.